File: module7/Collaboration/NodeJS/CreateRoom.js
var request = require("request");

var options = { method: 'POST',
  url: 'https://api.ciscospark.com/v1/rooms',
  headers: 
   { 'postman-token': '7ec3398f-f8c4-8c0b-c6a5-9fdc16af061b',
     'cache-control': 'no-cache',
     'content-type': 'application/json',
     authorization: 'Bearer NTIyN2U5NGQtY2JjZi00NGYxLWE0OTMtYzMwYTY3OWQzMDljOGE0YzA3ZTctNjY1' },
  body: { title: 'API Created Room' },
  json: true };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: module7/Collaboration/Python/CreateRoom.py
import requests

url = "https://api.ciscospark.com/v1/rooms"

payload = "{\n    \"title\": \"API Created Room\"\n}\n"
headers = {
    'authorization': "Bearer <your token>",
    'content-type': "application/json",
    'cache-control': "no-cache",
    'postman-token': "f3bb525f-f875-5e61-ab96-02472ae8dc11"
    }

response = requests.request("POST", url, data=payload, headers=headers)

print(response.text)